ui.tests/test-module/specs/fragment/fragment.runtime.spec.js

```diff
@@ -178,4 +178,20 @@ describe("Form Runtime with Fragment", () => {
                     });
             });
     });
+
+
+    it("rules inside fragment should work when included in form", () => {
+        const [idTextBox, fieldView] = Object.entries(formContainer._fields)[0];
+        const [idTextBox1, fieldView1] = Object.entries(formContainer._fields)[1];
+        const [idPanel, panelView] = Object.entries(formContainer._fields)[2];
+        const input = "abc";
+        const model = formContainer._model.getElement(idPanel);
+        cy.get(`#${idTextBox}`).find("input").clear().type(input).blur().then(x => {
+            cy.get(`#${idTextBox} input`).invoke('val').then(val => {
+                expect(val, "rules inside fragment (when included in form) did not work").to.equal(input);
+            })
+        })
+
+        cy.get(`#${model.items[0].id}`).should('have.text', 'Thanks');
+    });
```

Comment:
can we also add a test case where the same fragment is added multiple times (need to check if core components provide new id to the fields)

Reply:
Thanks. Added the same